hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -90,21 +90,21 @@ public ClientHeapNearCache(String mapName, ClientContext context, NearCacheConfi
         this.timeToLiveMillis = TimeUnit.SECONDS.toMillis(nearCacheConfig.getTimeToLiveSeconds());
         this.invalidateOnChange = nearCacheConfig.isInvalidateOnChange();
         this.evictionPolicy = EvictionPolicy.valueOf(nearCacheConfig.getEvictionPolicy());
-        this.canCleanUp = new AtomicBoolean(true);
+        this.canExpire = new AtomicBoolean(true);
```

Comment:
I would push everything that can be done in the 'class constructor' so that this constructor remains short. It also helps to quickly see which fields are just always set independent of the parameters, and which rely on the parameters. 

```
private final AtomicBoolean canExpire = new AtomicBoolean(true);
```